docs/README.md

```diff
@@ -0,0 +1,7 @@
+Elfpy documentation and notebook examples can be built locally using the provided Makefile. To build local documentations that include notebook examples in `examples/*_notebooks.py`, run from this directory
+
+```
+make notebook;
+make html
```

Comment:
what does `make html` run? I only see `notebook` in `docs/Makefile`

Reply:
`make html` gets passed into Sphinx here:
https://github.com/delvtech/elf-simulations/blob/7f46eb985dfc16f60a8c69d258635929966129d5/docs/Makefile#L20

This makefile is autogenerated from Sphinx, I just added a `notebook` check into this file